### PR TITLE
feat: Go Button on Alternate Base URL field

### LINKED ISSUE #425: Go button not working on alternate URL
Expected: 
1. Go button should work on alternate URL view too.
2. next button not working on password when default base URL is unchecked.

Working on it.

### PR DESCRIPTION
**Pull request by mejariamol**
- added data variable action of type runnable to auth url container layout
- changed parameter of ViewUtils.hideKeyboard to View
- added hideKeyboard call in data binding Go

fixes #425

### CODE REVIEW

**Comment by mejariamol** [comment]
@iamareebjamal @Shailesh351 please review.

File: app/src/main/res/layout/auth_url_container.xml [hunk 1]
```diff
@@ -1,8 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
-<layout xmlns:android="http://schemas.android.com/apk/res/android">
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
 
     <data>
         <import type="android.view.View" />
+
+        <variable
+            name="action"
```
Comment (iamareebjamal) [inline app/src/main/res/layout/auth_url_container.xml]: Where is this action bound?

Comment (mejariamol) [inline app/src/main/res/layout/auth_url_container.xml]: in signup and login layouts. in the `include` item

Comment (iamareebjamal) [inline app/src/main/res/layout/auth_url_container.xml]: I mean is there a counterpart in Java?

Comment (mejariamol) [inline app/src/main/res/layout/auth_url_container.xml]: No.

Comment (iamareebjamal) [inline app/src/main/res/layout/auth_url_container.xml]: Then why is it needed?

Comment (mejariamol) [inline app/src/main/res/layout/auth_url_container.xml]: Because it is used by signup and login both layouts. So both will bind different actions on go.

Comment (iamareebjamal) [inline app/src/main/res/layout/auth_url_container.xml]: OK

**Pull request merged**